src/Packetery/Module/Order/CarrierModal.php

```diff
@@ -15,6 +15,7 @@
 use Packetery\Module\Helper;
 use Packetery\Nette\Forms;
 use RuntimeException;
+use Packetery\Module\NativeCarrierSettingsConfig;
```

Comment:
Mně se na tom "native" nelíbí, že není jasné, pro co je to native. Pro náš plugin? Pro WP? Pro WC?
Navrhuji upřesnit.

Reply:
Prejmenovano.